py/DREAM/Output/EquationSystem.py

```diff
@@ -27,6 +27,7 @@ class EquationSystem:
         'j_tot':        CurrentDensity,
         'n_cold':       FluidQuantity,
         'n_hot':        FluidQuantity,
+        'N_i':          FluidQuantity,
```

Comment:
think I've commented this already, but I'm not sure if this (and W_i) should be fluidquantities, but perhaps rather some form of new ion quantity?

Reply:
Actually, this change made Ida's script crash with the following interface backtrace:

> Traceback (most recent call last):
> 
>   File "./run_scan2.py", line 340, in <module>
>     runiface(ds1,'out_1init.h5',quiet=True)
>   File "../../py/DREAM/runiface.py", line 73, in runiface
>     obj = DREAMOutput(outfile)
>   File "../../py/DREAM/DREAMOutput.py", line 44, in __init__
>     self.load(filename=filename, path=path)
>   File "../../py/DREAM/DREAMOutput.py", line 87, in load
>     self.eqsys = EquationSystem(od['eqsys'], grid=self.grid, output=self)
>   File "../../py/DREAM/Output/EquationSystem.py", line 57, in __init__
>     self.setUnknowns(unknowns)
>   File "../../py/DREAM/Output/EquationSystem.py", line 121, in setUnknowns
>     self.setUnknown(name=uqn, data=unknowns[uqn], attr=attr)
>   File "../../py/DREAM/Output/EquationSystem.py", line 101, in setUnknown
>     o = self.SPECIAL_TREATMENT[name](name=name, data=data, attr=attr, grid=self.grid, output=self.output)
>   File "../../py/DREAM/Output/FluidQuantity.py", line 27, in __init__
>     raise Exception("Unrecognized shape of data for '{}': {}. Expected (nt, nr) = ({}, {}).".format(name, data.shape, grid.t.size, grid.r.size))
> Exception: Unrecognized shape of data for 'N_i': (6, 3, 20). Expected (nt, nr) = (6, 20).
>